edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.adoc

```diff
@@ -72,13 +72,13 @@ include::modules/cnf-image-based-upgrade.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc#cnf-image-based-upgrade-for-sno[Performing an image-based upgrade with Lifecycle Agent]
+* xref:../../edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc#cnf-image-based-upgrade[Performing an image-based upgrade for {sno} clusters with {lcao}]
```

Comment:
```suggestion
* xref:../../edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc#cnf-image-based-upgrade_cnf-image-based-upgrade-base[Performing an image-based upgrade for {sno} clusters with {lcao}]
```

This would be the typical format so maybe a context variable is missing somewhere if this does not work.

Reply:
My bad, Context variable should be an ID. Thanks for reaching out on Slack, Alexandra.